src/actors/satellite-detector/satellite_detector.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+
+import requests
+import shlex
+
+from subprocess import Popen, PIPE
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+}
+
+DOCUMENTATION = '''
+---
+module: satellite_detector
+
+short_description: This module will detect satellite installation and its services
+
+
+description:
+    - "This module will detect satellite 5 or 6 installation and its respective services services"
+    - Output example:
+        10.10.10.21 | SUCCESS => {
+            "changed": false,
+            "failed": false,
+            "satellite_detected": true,
+            "satellite_services": [
+                "rh-postgresql95-postgresql",
+                "jabberd",
+                "tomcat6",
+                "httpd",
+                "osa-dispatcher",
+                "rhn-search",
+                "cobblerd",
+                "taskomatic"
+            ],
+            "satellite_version_bugfix": "0.31",
+            "satellite_version_major": "5",
+            "satellite_version_minor": "8"
+        }                                                                                                }
+    - Marcel Gazdik
+'''
+
+EXAMPLES = '''
+- name: Detect satellite installation
+  satellite_detector: ""
+  register: detection_result
+'''
+
+RETURN = '''
+satellite_detected:
+    description: A flag determining whether a satellite installation was detecgted
+    type: boolean
+
+satellite_services:
+    description: An array holding lost of services if satellite was found. This field is not
+                 not provided, if there is no satellite detected.
+    type: array of strings
+
+satellite_version_major:
+    description: An major version of found satellite instance. This field is not
+                 not provided, if there is no satellite detected.
+    type: string
+
+satellite_version_minor:
+    description: An minor version of found satellite instance. This field is not
+                 not provided, if there is no satellite detected.
+    type: string
+
+satellite_version_bugfix:
+    description: An bugfix version of found satellite instance. This field is not
+                 not provided, if there is no satellite detected.
+    type: string
+'''
+
+
+class DetectionFailed(RuntimeError):
+    pass
+
+
+def extract_version(version_string, result):
+    version_sep = version_string.split(".")
+    result["satellite_version_major"] = version_sep[0]
```

Comment:
hmm why do you split version to major, minor and bugfix? Do you have good reason for that?